superdesk/data_updates/00020_20191106-180807_archive.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8; -*-
+# This file is part of Superdesk.
+# For the full copyright and license information, please see the
+# AUTHORS and LICENSE files distributed with this source code, or
+# at https://www.sourcefabric.org/superdesk/license
+#
+# Author  : Jérôme
+# Creation: 2019-11-06 18:08
+
+from superdesk.commands.data_updates import DataUpdate
+from apps.archive.common import transtype_metadata
+from datetime import datetime
+
+
+class DataUpdate(DataUpdate):
+
+    resource = 'archive'
+
+    def forwards(self, mongodb_collection, mongodb_database):
+        for resource in ['archive', 'archive_autosave', 'published']:
```

Comment:
I think it would be good to check if there is any extra date field defined on instance. I think usually there won't be any but there might be lots of items with extra data like on stt where we migrate their archives and this upgrade would take there forever

Reply:
:+1:

Reply:
:+1: